src/main/java/uk/gov/justice/hmpps/casenotes/services/ExternalApiService.java

```diff
@@ -50,6 +55,27 @@
         return body;
     }
 
+    List<BookingIdentifier> getIdentifiersByBookingId(final Long bookingId) {
+        final var uri = clientCredentialsRestTemplate.getUriTemplateHandler().expand(new UriTemplate("/api/bookings/{bookingId}/identifiers").expand(bookingId).toString());
+
+        final var exchange = clientCredentialsRestTemplate.exchange(uri, HttpMethod.GET, null,
+                new ParameterizedTypeReference<List<BookingIdentifier>>() {
+                });
+        return exchange.getBody();
+    }
+
+    Optional<OffenderBooking> getBooking(final Long bookingId) {
+        final var uri = clientCredentialsRestTemplate.getUriTemplateHandler().expand(new UriTemplate("/api/bookings/{bookingId}?basicInfo=true").expand(bookingId).toString());
+        final var booking = new AtomicReference<Optional<OffenderBooking>>();
+        try {
+            final var exchange = clientCredentialsRestTemplate.exchange(uri, HttpMethod.GET, null, OffenderBooking.class);
+            booking.set(Optional.ofNullable(exchange.getBody()));
+        } catch (RestClientException e) {
+            booking.set(Optional.empty());
```

Comment:
feels a bit odd to ignore this, what is your thinking?

Reply:
In retrospect I'm now not sure, I think the exception should bubble up - it would imply that the booking id is not found.  I'll remove